libnavui-maps/src/main/java/com/mapbox/navigation/ui/maps/camera/data/MapboxNavigationViewportDataSource.kt

```diff
@@ -791,4 +790,8 @@ class MapboxNavigationViewportDataSource(
         debugger?.overviewPoints = pointsForOverview
         debugger?.overviewUserPadding = overviewPadding
     }
+
+    private fun Location.toPoint(): Point {
+        return Point.fromLngLat(this.longitude, this.latitude)
+    }
```

Comment:
Should we move this into `libnavigation-util`?

Reply:
Done.